src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -38,242 +39,247 @@ define([
-                var key = event.keyCode,
-                    $el = $(event.target);
+                viewKeypress: function(event) {
+                    var key = event.keyCode,
+                        $el = $(event.target);
 
-                if (_.contains(this.keyForward, key) || _.contains(this.keyBack, key)) {
-                    // Prevent default behavior if one of our trigger keys
-                    event.preventDefault();
-                }
+                    if (_.contains(this.keyForward, key) || _.contains(this.keyBack, key)) {
+                        // Prevent default behavior if one of our trigger keys
+                        event.preventDefault();
+                    }
 
-                if (key === constants.keyCodes.tab && !event.shiftKey && _.first($el) === _.first(this.$lastItem) ) {
-                    event.preventDefault();
-                    this.$dropdownButton.focus();
-                } else if (_.contains(this.keyClose, key)) {
-                    this.closeDropdownMenu();
-                    this.$dropdownButton.focus();
-                } else if ($el.hasClass('action')) {
-                    // Key handlers for when a menu item has focus
-                    this.handlerIsAction(key, $el);
-                } else if ($el.hasClass('dropdown-menu')) {
-                    // Key handlers for when the menu itself has focus, before an item within it receives focus
-                    this.handlerIsMenu(key);
-                } else if ($el.hasClass('has-dropdown')) {
-                    // Key handlers for when the button that opens the menu has focus
-                    this.handlerIsButton(key, event);
+                    if (key === constants.keyCodes.tab && !event.shiftKey && _.first($el) === _.first(this.$lastItem)) {
+                        event.preventDefault();
+                        this.$dropdownButton.focus();
+                    } else if (_.contains(this.keyClose, key)) {
+                        this.closeDropdownMenu();
+                        this.$dropdownButton.focus();
+                    } else if ($el.hasClass('action')) {
+                        // Key handlers for when a menu item has focus
+                        this.handlerIsAction(key, $el);
+                    } else if ($el.hasClass('dropdown-menu')) {
+                        // Key handlers for when the menu itself has focus, before an item within it receives focus
+                        this.handlerIsMenu(key);
+                    } else if ($el.hasClass('has-dropdown')) {
+                        // Key handlers for when the button that opens the menu has focus
+                        this.handlerIsButton(key, event);
+                    }
                 }
-            }
-        });
-
-        return DropdownMenuView;
-    }
+            });
+
+            return DropdownMenuView;
+        }
+    );
+}).call(
+    this,
+    (typeof define === 'function' && define.amd && define) ||
```

Comment:
Also, isn't that last `&& define` redundant with `typeof define === 'function'`?